runtime/tests/port/j9vmemTest.c

```diff
@@ -2165,18 +2173,32 @@ j9vmem_testFindValidPageSize_impl(struct J9PortLibrary *portLibrary, char *testN
 			sixteenMBPageSize = TRUE;
 		}
 
+		if ((2*ONE_MB == pageSizes[i]) &&
+			(0 != (J9PORT_VMEM_PAGE_FLAG_NOT_USED & pageFlags[i]))
+		) {
+			twoMBPageSize = TRUE;
+		}
+
 		if ((16*FOUR_KB == pageSizes[i]) &&
 			(0 != (J9PORT_VMEM_PAGE_FLAG_NOT_USED & pageFlags[i]))
 		) {
 			sixtyFourKBPageSize = TRUE;
 		}
+
+		if ((FOUR_KB == pageSizes[i]) &&
+			(0 != (J9PORT_VMEM_PAGE_FLAG_NOT_USED & pageFlags[i]))
```

Comment:
Consider using `J9_ARE_ANY_BITS_SET` to improve readability. Additionally, the `&&` operator should be placed at the beginning of the next line, as per the [coding standard](https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md#formatting-conditions). There are a few other instances of incorrect && placement in this code that should be corrected for consistency.
```suggestion
		if ((FOUR_KB == pageSizes[i])
			&& J9_ARE_ANY_BITS_SET(pageFlags[i], J9PORT_VMEM_PAGE_FLAG_NOT_USED)
```

Reply:
Updated.